packages/renderer/src/modules/store/translation/locales/es.json

```diff
@@ -22,26 +22,23 @@
   "scene_list": {
     "my_scenes": "Mis Escenas",
     "import_scene": "Importar Escena",
-    "create_scene": "Crear Escena",
+    "templates": "Plantillas",
+    "sort_by": "Ordenar por",
     "project_actions": {
       "delete_project": "Borrar de Mis Escenas",
       "duplicate_project": "Duplicar",
       "open_folder": "Abrir Carpeta"
     },
     "no_scenes": {
       "title": "Crea tu primera escena",
-      "description": "Desata tu creatividad. Comienza a construir escenas para sus tierras y mundos y comparta con la comunidad.<a> Aprenda más sobre cómo crear escenas. </a>",
-      "from_scratch": {
-        "title": "Empezar desde el principio",
-        "description": "Comience a construir sus escenas desde cero con el constructor en línea de una manera simple e intuitiva."
-      }
+      "description": "Desata tu creatividad. Comienza a construir escenas para sus tierras y mundos y comparta con la comunidad.<a> Aprenda más sobre cómo crear escenas. </a>"
```

Comment:
`Comienza a construir escenas para sus tierras y mundos y comparta con la comunidad` see,s weird texting. Maybe it should be `Comience ...` or even better `Comienza a construir escenas para tus tierras y mundos...`